src/main/java/java/lang/String.java

```diff
@@ -3946,9 +3987,23 @@ public String toString() {
      * @diffblue.untested
      */
     public static String format(String format, Object... args) {
-        // DIFFBLUE MODEL LIBRARY This is treated internally in CBMC
-        return CProver.nondetWithoutNullForNotModelled();
+        // DIFFBLUE MODEL LIBRARY
```

Comment:
:pick: I think the new standard is to put the commented out JDK code first, and then `// DIFFBLUE MODEL LIBRARY`.